transport-native-unix-common/src/main/java/io/netty/channel/unix/Socket.java

```diff
@@ -298,7 +298,7 @@ public int send(ByteBuffer buf, int pos, int limit) throws IOException {
         return ioResult("send", res);
     }
 
-    public int sendAddress(long address, int pos, int limit) throws IOException {
+    public int sendAddress(jobject address, int pos, int limit) throws IOException {
```

Comment:
why jobject?

Reply:
I think I might have hit the wrong vim macro there ...